README.md

```diff
@@ -1,2 +1,87 @@
+    - Testing requirements (e.g. is demonstrated to run a defined piece of software on a test robot without fault, with a certain response time for libkoki markers, etc.)
+- The _Dev Team_ may also define some limits, such as:
+    - Cut-off date for deployment at SR2020
+    - Cost per unit limit
+    - Evidence shown that more than one board has been considered
+- The _Dev Team_ may consult the _Core Team_ or Trustees to assist them with requirements. For example, in the case of a Project that would have a high implementation cost, the _Dev Team_ would need to communicate the feasibility of it with the Trustees.
+- This specification is presented to and ratified by the _Dev Team_ (either on the _Dev Team_ mailing list or at a _Dev Team_ meeting)
+- Development work happens.
+- If changes to the specification are needed, these can be discussed with the _Dev Team_.
+- Development reaches a point where the project group is happy the work is done and is shippable.
+- _Dev Team_ reviews the equipment against the agreed specification and decides whether to recommend it to the _Core Team_.
+- The process is now passed onto the _Core Team_ who are responsible for deciding when and how to deploy it (e.g for SR 2020).
+    - If it meets the needs of the competition, it is the _Core Team_’s responsibility to ensure new kit is purchased, tested and deployed (although it’s likely to be the _project group_ who do this).
+    - If it doesn’t, the _Core Team_ must decide whether to proceed anyway at risk, or to continue with the existing kit and pause expansion.
+
+### Project of minor impact: Addition to the API (e.g. making an “R.ruggeduinos[0].PWM()” )
+
+As this is notionally a simpler thing, but still in scope as it impacts the competition cycle, the process can be simpler.
+
+- Developer wants to add the feature and contacts the appropriate _project group_ or creates one if it does not exist.
+- The _project group_ determines that there is no need for a specification due to minor impact.
+    - A change of minor impact does not have significant cost or change to the working of the competition.
+    - The _Dev Team_ may determine that a project is of minor impact and require a specification.
+- Developer updates/opens a new ticket in the appropriate GitHub Project.
+- The _project group_ will define testing and documentation requirements.
+- Developer submits a pull request. 
+- The _project group_ checks it meets the requirements and approves or rejects.
+- The _Dev Team_ will review the new version and decides whether to recommend it to the _Core Team_.
+- Software may be continuously developed throughout the competition, but it only deployed at the discretion of the _Core Team_. The _Core Team_ may choose to not upgrade software or equipment if it is not deemed beneficial.
+- When deploying, the _Core Team_ will update the kit to use the latest stable version of the software.
```

Comment:
It's highly unlikely that the core team be the ones updating the kit. Once the core team give the green light, it'll be the reponsibility of the dev team / project group to ensure everything is in a shippable state, the core team to allocate someone to deploy it (this may be someone in the dev team / project group), and then it's shipped :shipit:

Reply:
In my opinion the _Dev Team_ should be constantly working on things and mark versions as stable. The Core Team will then deploy that version during prep for Kickstart etc.

Reply:
This makes it unrealistic for, say, one very good software developer to update the kit. Let's say Steve the Software Developer writes some profoundly good firmware from the camera board as the only member of the project group. He must then deploy all the instances of this software, which if we have a lot of camera boards is an absurd quantity of work for one person.

In general, this appears to serve to massively discourage small group development, because that small group will then be incapable of deploying whatever they've developed, even though the fruits of their labour are perfectly good code / hardware.

Reply:
That's true, perhaps the project group should work with the core team to ensure shipment happens correctly? Perhaps also by suggesting those who may be able to assist. 

Related to https://github.com/srobo/dev-team-proposal/pull/1#discussion_r261861887

Reply:
In that instance the Core Team would ask for the project group to help them.

Nowhere does it say that the project groups cannot work directly with the core team if deemed necessary

Reply:
> In that instance the Core Team would ask for the project group to help them.

We should define that in this example, as it's an important distinction.

Reply:
Suggested wording?

Reply:
```suggestion
- When deploying, the _Core Team_ will ensure the kit is updated with the newly created software, with help from the relevant _project group_.
```